files/workflows/ZWEWRF04.properties

```diff
@@ -0,0 +1,33 @@
+# zowe_build
+# Label: Zowe Build
+# Abstract: Path to the extracted convenience build
+# Category: Build Location
+# Description:
+#  Path to the folder from the extracted convenience build PAX file.
+zowe_build=/u/users/ibmusers/zowe-build/zowe-1.8.0
```

Comment:
the path is more likely going to be `~/zowe-1.9.0`
`/u/users/ibmusers/zowe-build` sounds like something very specific to your environment